manim/mobject/coordinate_systems.py

```diff
@@ -710,6 +717,482 @@ def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+            The first (i.e the non faded lines andd circles) and second (i.e the faded lines and circles) sets of lines and circles, respectively.
+        """
+        center = self.get_center_point()
+        ratio_faded_lines = self.faded_line_ratio
+        offset = self.azimuth_offset
+
+        if ratio_faded_lines == 0:  # don't show faded lines
+            ratio_faded_lines = 1  # i.e. set ratio to 1
+        rstep = (1 / ratio_faded_lines) * self.x_axis.x_step
+        astep = (1 / ratio_faded_lines) * (TAU * (1 / self.azimuth_step))
+        rlines1 = VGroup()
+        rlines2 = VGroup()
+        alines1 = VGroup()
+        alines2 = VGroup()
+
+        rinput = np.arange(0, self.x_axis.x_max + rstep, rstep)
+        ainput = np.arange(0, TAU, astep)
+
+        unit_vector = self.x_axis.get_unit_vector()[0]
+
+        for k, x in enumerate(rinput):
+            new_line = Circle(radius=x * unit_vector)
+            if k % ratio_faded_lines == 0:
+                alines1.add(new_line)
+            else:
+                alines2.add(new_line)
+
+        line = Line(center, self.get_x_axis().get_end())
+
+        for k, x in enumerate(ainput):
+            new_line = line.copy()
+            new_line.rotate(x + offset, about_point=center)
+            if k % ratio_faded_lines == 0:
+                rlines1.add(new_line)
+            else:
+                rlines2.add(new_line)
+
+        lines1 = VGroup(*rlines1, *alines1)
+        lines2 = VGroup(*rlines2, *alines2)
+        return lines1, lines2
+
+    def get_center_point(self):
+        return self.coords_to_point(0, 0)
+
+    def get_x_unit_size(self):
+        return self.get_x_axis().get_unit_size()
+
+    def get_y_unit_size(self):
+        return self.get_x_axis().get_unit_size()
+
+    def get_axes(self) -> VGroup:
+        """Gets the axes.
+        Returns
+        -------
+        :class:`~.VGroup`
+            A pair of axes.
+        """
+        return self.axes
+
+    def get_vector(self, coords, **kwargs):
+        kwargs["buff"] = 0
+        return Arrow(
+            self.coords_to_point(0, 0), self.coords_to_point(*coords), **kwargs
+        )
+
+    def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        for mob in self.family_members_with_points():
+            num_curves = mob.get_num_curves()
+            if num_inserted_curves > num_curves:
+                mob.insert_n_curves(num_inserted_curves - num_curves)
+        return self
+
+    def polar_to_point(self, radius: float, azimuth: float) -> np.ndarray:
+        r"""Gets a point from polar coordinates.
+
+        Parameters
+        ----------
+        radius
+            The coordinate radius (:math:`r`).
+
+        azimuth
+            The coordinate azimuth (:math:`\theta`).
+
+        Returns
+        -------
+        numpy.ndarray
+            The point.
+
+        Examples
+        --------
+
+        .. manim:: PolarToPointExample
+            :ref_classes: PolarPlane
```

Comment:
For the sake of completeness
```suggestion
            :ref_classes: PolarPlane Vector
```